src/config/config.cc

```diff
@@ -159,6 +159,8 @@ Config::Config() {
        new IntField(&force_compact_file_min_deleted_percentage, 10, 1, 100)},
       {"db-name", true, new StringField(&db_name, "change.me.db")},
       {"dir", true, new StringField(&dir, kDefaultDir)},
+      {"snapshot-dir", false, new StringField(&snapshot_dir, "")},
+      {"keep-snapshot", false, new YesNoField(&keep_snapshot, false)},
```

Comment:
We can add them into `kvrocks.conf`, and add some comments for these new options.